Sources/App/Core/Rollbar.swift

```diff
@@ -49,15 +49,15 @@ extension Rollbar {
         }
 
         struct Data: Content {
-            let environment: String
-            let body: Body
-            let level: Level
-            let language = "swift"
-            let framework = "vapor"
-            let uuid = UUID().uuidString
+            var environment: String
+            var body: Body
+            var level: Level
+            var language = "swift"
+            var framework = "vapor"
+            var uuid = UUID().uuidString
```

Comment:
What changed in Xcode/Swift to make this necessary? (Just curious more than anything!)

Reply:
That was actually a useful warning (it didn't error, just warned): The issue it flagged was that we declared a type `Decodable` but then had `let language = "swift"` - which makes it impossible for decoding to actually set `language`.